server/src/main/java/com/google/codeu/servlets/RestaurantServlet.java

```diff
@@ -44,8 +49,38 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       response.sendRedirect("/feed");
       return;
     }
-    Restaurant restaurant = new Restaurant(name, address, bio);
+    // Find the coords of the address
+    Double[] coord = new Double[2];
+    try {
+      coord = this.lookupCoord(address);
+    } catch (ApiException a) {
+      response.sendRedirect("/feed");
```

Comment:
nit: You can catch multiple exceptions in one catch.

`catch (ApiException | InterruptedException | IoException e)`